src/api/initialDataTypes.js

```diff
@@ -181,8 +182,11 @@ export type InitialDataRealm = $ReadOnly<{|
   realm_bot_creation_policy: number,
   realm_bot_domain: string,
 
-  // TODO(server-3.0): Added in feat. 11
-  realm_community_topic_editing_limit_seconds?: number,
+  // TODO(server-8.0): Added in feat. 225
+  realm_can_access_all_users_group?: boolean,
+
+  // TODO(server-8.0): Added in feat. 209
+  realm_create_multiuse_invite_group?: number,
 
```

Comment:
Reading through the changelog, here's a small other item at FL 195:
> [GET /events](https://zulip.com/api/get-events), [POST /register](https://zulip.com/api/register-queue): The default_code_block_language realm setting is now consistently an empty string when no default pygments language code is set. Previously, the server had a bug that meant it might represent no default for this realm setting as either null or an empty string. Clients supporting older server versions should treat either value (null or "") as no default being set.

So the line ` realm_default_code_block_language: string | null,` should get a `TODO(server-8.0)` comment saying it's just `string` in current versions.